corehq/apps/locations/tests/test_reupholster.py

```diff
@@ -0,0 +1,87 @@
+from ..models import Location, SQLLocation
+from .test_locations import LocationTestBase
+from .util import make_loc
+
+
+class TestReupholster(LocationTestBase):
+    """
+    These tests were written to drive removal of sepecific queries. It
+    is safe to delete this when the reuholstering of Location is done
+    and somone has written test coverage for the methods used in here.
+    """
+    def setUp(self):
+        super(TestReupholster, self).setUp()
+        self.state = make_loc("Florida", type='state')
+        self.district = make_loc("Duval", type='district', parent=self.state)
+        self.block = make_loc("Jacksonville", type='block', parent=self.district)
+
+    def test_replace_all_ids(self):
+        original_result = set([r['id'] for r in Location.get_db().view(
+            'locations/by_name',
+            reduce=False,
+        ).all()])
+
+        new_result = set(SQLLocation.objects.ids())
+
+        self.assertEqual(original_result, new_result)
+
+    def test_all_for_domain_by_type(self):
+        original_result = set([r['id'] for r in Location.get_db().view(
```

Comment:
not totally sure i understand the intention of this test. it seems like it would be more useful if this this called a function that lives in `dbaccessors` as opposed to directly querying a couch view (and then whatever code it's supposed to be testing can also call that same `dbaccessors` function).